integration/__snapshots__/lerna-bootstrap.test.js.snap

```diff
@@ -1,25 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`lerna bootstrap --npm-client yarn: stderr 1`] = `
-lerna info version __TEST_VERSION__
-lerna info Bootstrapping 4 packages
-lerna info lifecycle package-4@1.0.0~preinstall: package-4@1.0.0
-lerna info Installing external dependencies
-lerna info Symlinking packages and binaries
-lerna info lifecycle @integration/package-3@1.0.0~postinstall: @integration/package-3@1.0.0
-lerna info lifecycle @integration/package-2@1.0.0~prepublish: @integration/package-2@1.0.0
-lerna info lifecycle @integration/package-1@1.0.0~prepare: @integration/package-1@1.0.0
-lerna success Bootstrapped 4 packages
-`;
-
-exports[`lerna bootstrap --npm-client yarn: stdout 1`] = `
-package-1
-package-2
-cli package-2 OK
-package-3 cli1 OK
-package-3 cli2 package-2 OK
-`;
-
```

Comment:
I think this change is not related to my work, but it's needed to make `npm run ci` pass.